cypress/e2e/supervision/clients/documents/open-document.cy.js

```diff
@@ -0,0 +1,68 @@
+import path from "path";
+
+beforeEach(() => {
+  cy.loginAs("Allocations User");
+  cy.createAClient();
+  cy.loginAs("Public API");
+  cy.uploadDocument();
+});
+
+describe(
+  "Open document successfully",
+  { tags: ["@supervision-core", "@documents", "@open-document", "@smoke-journey"] },
+  () => {
+    it("opens a document successfully", () => {
+      cy.loginAs("Case Manager");
+      cy.get("@clientId").then((clientId) => {
+        cy.visit(`/supervision/#/clients/${clientId}`);
+        cy.contains("Ted Tedson");
+      });
+
+      cy.get(".TABS_DOCUMENTS").click();
```

Comment:
You may want to move this inside the then above to a void race conditions due to the way Cypress queues commands rather than executes them inside the test. i.e. this might run before you have visited the client page.